styles/components/Footer.module.scss

```diff
@@ -19,6 +19,10 @@
     @apply flex gap-4;
     @apply items-center;
   }
+  .socialButtons,
+  .logo {
+    @apply max-[500px]:hidden;
+  }
 }
```

Comment:
the logo and buttons should probably be the one to stay and the email should be the one to go